tests/components/webostv/test_config_flow.py

```diff
@@ -149,8 +149,30 @@ async def test_form(hass, client):
     assert result["title"] == TV_NAME
 
 
-async def test_options_flow(hass, client):
-    """Test options config flow."""
+@pytest.mark.parametrize(
+    "apps, inputs",
+    [
+        # Live TV in apps (default)
+        (None, None),
+        # Live TV in inputs
+        (
+            {},
+            {
+                "in1": {"label": "Input01", "id": "in1", "appId": "app0"},
+                "in2": {"label": "Input02", "id": "in2", "appId": "app1"},
+                "livetv": {"label": "Live TV", "id": "livetv", "appId": LIVE_TV_APP_ID},
+            },
+        ),
+        # Live TV not found
+        ({}, None),
+    ],
+)
+async def test_options_flow_live_tv_in_apps(hass, client, apps, inputs):
+    """Test options config flow Live TV found in apps."""
+    if apps is not None:
+        client.apps = apps
```

Comment:
What will `client.apps` return if we don't set it here?

Reply:
The default used in other tests has Live TV app in it:
https://github.com/home-assistant/core/blob/77ef86faee0c7eac3a418dc5784a0aa9d2c769c7/tests/components/webostv/conftest.py#L31-L37

This test has 3 stages:
1. Doesn't change the fixture so that Live TV is found in apps (this is the default behavior)
2. Set empty apps and add Live TV to input sources to test Live TV is found in input sources.
3. Set empty apps but does not add Live TV to input sources - test case for Live TV missing (in this case we just add dummy Live TV input).

The reason 1 & 2 needed is that on some devices the Live TV source is not named "Live TV" (on my device for example it is named "Idan+" which is the local broadcast TV name), but app id doesn't change in this case

Reply:
Can we set the default too so we don't need conditional logic in the test?

Reply:
I apologize for many changed files, had to move test constants to a separate file else imports becomes messy.